scripts/pre-commit.sh

```diff
@@ -29,6 +29,26 @@ run_swiftlint() {
   fi
 }
 
+verify_no_included_apikeys() {
+  SCRIPT_DIR=$(cd $(dirname $(readlink -f "${BASH_SOURCE[0]}")) && pwd)
```

Comment:
`readlink -f` follows the link from `.git/hooks/pre-commit` back to `scripts/pre-commit.sh` if executed through there. If not it returns the same file.

Reply:
maybe we can add that as a comment?

Reply:
Done.